app/services/moderator/banish_user.rb

```diff
@@ -46,7 +46,7 @@ def remove_profile_info
         twitch_url: nil, feed_url: nil
       )
 
-      user.update_columns(profile_image: "https://thepracticaldev.s3.amazonaws.com/i/99mvlsfu5tfj9m7ku25d.png")
+      user.update_columns(profile_image: "https://#{ApplicationConfig['AWS_BUCKET_NAME']}.s3.amazonaws.com/i/99mvlsfu5tfj9m7ku25d.png")
```

Comment:
this is further evidence we might want to rethink this approach, we are storing a broken path in the DB:

- it's either a broken URL for new installers
- or a URL that points to non existent images if the bucket name is not `thepracticaldev`